proprietary/Components/src/denhaag/highlighted-links.tokens.json

```diff
@@ -0,0 +1,26 @@
+{
+  "denhaag": {
+    "highlighted-links": {
+      "background-color": { "value": "{denhaag.color.warmgrey.1}" },
+      "padding-block-start": { "value": "{denhaag.space.block.3xl}" },
+      "padding-block-end": { "value": "{denhaag.space.block.3xl}" },
+      "width": { "value": "100%" },
+      "container": {
+        "margin-inline-start": { "value": "auto" },
+        "margin-inline-end": { "value": "auto" },
+        "width": { "value": "100%" },
+        "max-width": { "value": "98rem" }
```

Comment:
If this component is within a parent component like the page-container, this property redundant. Will this component used outside or inside a parent component? Otherwise there will be 2 containers colliding.